Certification-Standard/introduction.md

```diff
@@ -0,0 +1,239 @@
+*RSoulat* Could be Auditors, or any "certificate provider", the difference with the CC is that the evaluation authority is **not** the certificate issuer
+
+**Exploitable vulnerability**: Weakness in the Target of Evaluation (TOE) that can be used to violate the Security Functional Requirement (SFRs) in the Operation Environment (OE) for the TOE
+
+**Object**: Entity in the Target of Evaluation that contains or receives information and upon which subjects perform operations.
+*RSoulat* Maybe we could also augment this definition to include UTxOs? 
+
+**Operation Environment**: Enfironment in which the Target of Evaluation (TOE) is operated, consisting of everything that is outside the TOE boundary
+*RSoulat* For example, the ledger, the other contracts, the off-chain code if the TOE is only the on-chain component
+
+**Potential vulnerability**: Suspected, but not confirmed, weakness
+
+**Protection Profile (PP)**: Implementation-independant statement of the security needs for a Target of Evaluation (TOE) type
+
+**Residual Vulnerability**: Weakness that cannot be exploited in the Operational Environment for the Target of Evaluation but that can be used to violate a Security Functional Requirement (SFR) by an attacker with greater Attack Potential than is anticipated in the operational environment for the TOE.
+
+**Security Assurance Requirements**: Security requirements that refers to the conditions and processes for the development and delivery of the Target of Evaluation (TOE) and the actions required of evaluators with respect to evidence produced from these conditions and processes.
+
+**Security Functional Requirements**: Security requirements, which contributes to fulfil the Target of Evaluation (TOE) Security Problem Definition (SPD) as defined in a specific Security Target (ST) or in a Protection Profile (PP).
+
+**Security Objective**: Statement of an intent to counter identified threats and/or identified organization security policies and/or assumptions.
+*RSoulat* Maybe overkill, maybe we could just have "counter identified threats" and not the rest. Even though I think organization security policies are a big component of security even for a DApp.
+
+**Security Problem Definition**: Statement, which in a formal manner defines the nature and scope of the security that the Target of Evaluation (TOE) is intended to address
+*RSoulat* I think this is one of the main thing we need to address
+
+**Security Requirement**: Requirement, which is part of a Target of Evaluation (TOE) security specification as defined in a specific Security Target (ST) or in a Protection Profile (PP)
+
+**Security Target**: Implementation-dependant statement of security requirements for a Target of Evaluation (TOE) based on a Security Problem Definition (SPD)
+
+**Subject**: Entity in the Target of Evaluation (TOE) that performs operations on objects
+
+**Target of Evaluation**: Set of software which is the subject of an evaluation
```

Comment:
Software and services?

Reply:
What would be a service that is not software?